lib/Serialization/Serialization.cpp

```diff
@@ -3726,10 +3726,28 @@ static void writeDeclCommentTable(
       if (!VD)
         return true;
 
-      // Skip the decl if it does not have a comment.
       RawComment Raw = VD->getRawComment();
-      if (Raw.Comments.empty() && !GroupContext.isEnable())
-        return true;
+
+      //when building the stdlib we intend to
```

Comment:
Nitpick: Please follow the comment style used by the rest of the file: capital letters, space after the `//`.